plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java

```diff
@@ -71,9 +75,11 @@ public TextTagBase() {
         // Returns a special chat code that makes the following text execute the input command line value when clicked.
         // To execute a command "/" should be used at the start. Otherwise, it will display as chat.
         // This tag must be followed by an <&end_click> tag.
-        // For example: - narrate "You can <&click[wow]>click here<&end_click> to say wow!"
-        // For example: - narrate "You can <&click[/help]>click here<&end_click> for help!"
         // Note that this is a magic Denizen tool - refer to <@link language Denizen Text Formatting>.
+        // @example
+        // - narrate "You can <&click[wow]>click here<&end_click> to say wow!"

```

Comment:
same as above, examples should have `# comments` on them noting what the usage does in practice